test/test_jit_fuser_te.py

```diff
@@ -490,7 +490,8 @@ def apply(fn):
             try:
                 t = torch.jit.trace(fn, (x, y, z))
                 self.assertEqual(ref, t(x, y, z))
-                self.assertAllFused(t.graph_for(x, y, z))
+                if dtype != torch.uint8:
```

Comment:
can we move the dtypes to `self.dtypes` and exclude `torch.uint8` ?

Reply:
Oh yeah that's a better idea than this.  I had kind of wanted to test a few uint cases just to make sure nothing was broken but no reason to exhaustively go through them here.